Documentation/Signoffs/Sign.md

```diff
@@ -0,0 +1,200 @@
+
+## Constraints
+
+1. Seven-Segment Display & Controls
+- Must have waterproof controllable display directly viewable in the daylight (1000 nits[8]) and from the road capable of showing number of open parking spots 
+- Must have a microcontroller with at least a 14 Digital output PINs, RX/TX PINs, and capable of connecting a external wireless communication module to connect to the 
+- Must have a power circuit rated for switching display LEDs on/off using microcontroller output PIN voltage and power from separate subsystem
+
+2. Outdoor Physical Sign/Stand
+- Must be large enough to hold controllable display and static text viewable from the road
+- Must be able to withstand reasonable weather requirements (storm, mid to heavy wind, etc.)
+
+3. Weatherproofing
+- Must protect Seven-Segment display controls from dust and water (Note: This design will be handled by the sign power subsystem)
+
+
+
+
+## Buildable Schematic
+
+![Figure 1. Arduino Circuit Diagram](../Electrical/Schematics/Sources/Display_Schematic.PNG)
+<div align="center"> Figure 1. Arduino Circuit Diagram
+<br />
+<div align="left">
+
+
+![Figure 2. Physical Sign Schematic](../3D&#32;Models/NewSignView1.PNG)
+<div align="center"> Figure 2. Physical Sign PDF Schematic 
+<br />
+<div align="left">
+
+
+![Figure 3. Physical Sign Schematic View 2](../3D&#32;Models/NewSignView2.PNG)
+<div align="center"> Figure 3. Physical Sign PDF Schematic View 2
+<br />
+<div align="left">
+
+
+## Analysis
+
+1. Seven-Segment Display & Controls
+
+a. LED Strips 
+
+- Will consist of 7 red 12 V LED light strips per display (14 total)
+- Wires will be fed through the sign through drilled holes at the end of each strip
+> - Each LED strip is roughly 6 inches long
+> - Each segment is roughly 7 inches (W) x 13 inches (H) 
+> - The following analysis shows how these seven-segments will meet the required constraints of visibility:
+> > - Length of road parallel to main entrance is roughly 100 feet and width is roughly 20 feet [6]
+> > - Maximum viewing distance when entering the lot is 102 feet [6]
+> > - Optimum viewing distance is 120 feet and maximum viewing distance is 525 feet [5]
+
+- Keiurot LED strips chosen for:
+> - Already waterproof (IP67 rated)
+> - LED strips rated for 1000 Lumens each, a max viewing apex angle of 160 degrees (average max for LED strips) equates to roughly 318.3 Candela. Over both seven-segment displays (16 inches x 13 inches) equate to 2,371.5 Nits (Cd/m^2) which meets the constraint of at least 1000 Nits total for daylight visibility. 
```

Comment:
The article is about displays. You don't have candela per square meter as this is individual LEDs.

Reply:
So, I'm still not convinced this will be visible.